pages_builder/pages/browse-list.yml

```diff
@@ -26,31 +26,42 @@ examples:
   -
     items:
       -
+        title: Apply to G-Cloud 11
+        link: '#nope'
+        statuses:
+          -
+            type: quiet
+            title: You haven’t started this yet
+      -
         title: Apply to provide teams for digital outcomes
         link: '#nope'
         body: Provide teams to build and support digital outcomes.
         statuses:
           -
-            message: You have completed this service.
-            important: True
+            type: happy
```

Comment:
In the generated patterns the `title` ends up at the bottom of the lists for some reason, which isn't as nice as in the YAML here where the title is at the top, e.g.: 
![screen shot 2015-11-16 at 08 49 42](https://cloud.githubusercontent.com/assets/6525554/11177698/4ceba15a-8c3f-11e5-9e6e-8f150bc71965.png)

It's not the end of the world, but not quite as clear what's going on.